mungegithub/deployment/README.md

```diff
@@ -41,7 +41,10 @@ We are using one basic load-balancer for each Submit Queue.
 * Auth: http://35.197.10.29:8080
 * Istio: http://35.197.104.17:8080
 * Mixer: http://35.197.95.47:8080
-* Pilot: http://35.199.174.118:8080 (Current disabled)
+* Pilot: http://35.199.174.118:8080
+* Test-Infra: http://104.196.250.254:8080
+* Mixerclient: http://35.199.147.28:8080
+* Proxy: http://35.197.99.161:8080
```

Comment:
We have more repos in istio, what about others? e.g. istio/api

Reply:
Right, we haven't cover all repos because usually they don't have as many PRs as those main repos. And like "api", we don't do auto-dependency-update, so it's not that necessary. But in unifying experience perspective, we should.